src/jit/gtlist.h

```diff
@@ -166,9 +169,14 @@ GTNODE(ADD_LO     , "+Lo"          ,1,GTK_BINOP)
 GTNODE(ADD_HI     , "+Hi"          ,1,GTK_BINOP)
 GTNODE(SUB_LO     , "-Lo"          ,0,GTK_BINOP)
 GTNODE(SUB_HI     , "-Hi"          ,0,GTK_BINOP)
-GTNODE(MUL_HI     , "*Hi"          ,1,GTK_BINOP)
 GTNODE(DIV_HI     , "/Hi"          ,0,GTK_BINOP)
 GTNODE(MOD_HI     , "%Hi"          ,0,GTK_BINOP)
+GTNODE(MUL_LONG   , "*long"        ,1,GTK_BINOP) // A mul that returns the 2N bit result of an NxN multiply. This op
+                                                 // is used for x86 multiplies that take two ints and return a long
+                                                 // result. All other multiplies with long results are morphed into
+                                                 // helper calls. It is similar to GT_MULHI, the difference being that
+                                                 // GT_MULHI drops the lo part of the result, whereas GT_MUL_LONG keeps
+                                                 // both parts of the result.
 #endif // !defined(LEGACY_BACKEND) && !defined(_TARGET_64BIT_)
```

Comment:
Not simply because of your change, but the comment heading this section is now quite out-of-date.